### ISSUE TITLE
ComfyUI support?

### ISSUE THREAD

**Issue by Pythonpa** [JackAILab/ConsistentID#25]
Do you have any plans to launch comfyui with official native support? Integrate into the ComfyUI ecosystem like IPAdapter or InstantID?

**Comment by JackAILab** [comment]
> Do you have any plans to launch comfyui with official native support? Integrate into the ComfyUI ecosystem like IPAdapter or InstantID?

Yes, we will also release the ComfyUI version, which has a lower priority than other tasks. We will release it in time, please stay tuned. If you have any further ideas, please feel free to ask~

**Comment by fatpandaria** [comment]
any progress?
